src/parser/name_resolution.c

```diff
@@ -11449,7 +11449,7 @@ pt_get_column_name_pre (PARSER_CONTEXT * parser, PT_NODE * node, void *arg, int
 
   if (node->node_type == PT_SELECT)
     {
-      *continue_walk = PT_STOP_WALK;
+      ;				//*continue_walk = PT_STOP_WALK;
```

Comment:
Why is it commented out?

Reply:
select (select id from t1@localsrv where v1=v2) from t2@localsrv;
In a query like the one in the example above, the select list could have a new select statement, so it needed to continue uninterrupted.

In this case, the if statement is empty, so let's clean it up.